css/main.css

```diff
@@ -268,30 +283,21 @@ ul {
   padding: 20;
 }
 
-.mother-ul {
-  font-size: 30px;
-  font-weight: bold;
-  display: contents;
-}
-
 .section4 {
-  background-color: #6070ff;
-  border-top-left-radius: 7rem;
-  padding-bottom: 250px;
-  margin-left: -39px;
-  margin-right: -8px;
-  margin-top: 50px;
-  margin-bottom: -60px;
+  background-color: white;
+  width: 100%;
 }
 
 .contact-us {
-  margin-top: 30px;
+  background-color: #6070ff;
+  border-top-left-radius: 7rem;
   padding-left: 0;
   padding-left: 10px;
   background-image: url("../image/contact-us.png");
   background-repeat: no-repeat;
   background-size: cover;
   background-position: right;
+  padding-bottom: 20px;
 }
 
 .contact-header > .c-header {
```

Comment:
- [ ] Since you have made improvements since the last review, there is still a matter that needs to be resolved in order for your project appear professional. If you need any suggestions, I may suggest you adjust the 'position' of your background image to properly position those icons on the background image. Kindly fix the 'background-image' and fix it as the one for the Figma design. 😉 

Your Design: 👇 
![yourbackground](https://user-images.githubusercontent.com/88894525/192767604-22601d2e-9bab-45dc-827a-bdb63b37679c.png)

Figma Design 👇 
![fixthebackground](https://user-images.githubusercontent.com/88894525/192767752-904e3739-e674-4003-bc17-bdca48629e38.png)